Tools/AP_Periph/networking.h

```diff
@@ -1,9 +1,9 @@
 #pragma once
 
-#include "AP_Periph.h"
-
 #ifdef HAL_PERIPH_ENABLE_NETWORKING
```

Comment:
We need to include *something* before this or we're dependent on include order whereever this is included.

Reply:
yup. Added <AP_HAL/AP_HAL_Boards.h>